ui/js/controllers/jobs.js

```diff
@@ -71,6 +71,10 @@ treeherderApp.controller('JobsCtrl', [
             }
         };
 
+        if($location.search().revision === 'undefined'){
```

Comment:
> There is a code, just below this code, that uses if without the spaces, should I modify it too??

My take anyway :) generally only if it's directly related to the change involved (ie. there's code changing on that line or within that sub-block). It helps not confuse the reviewer, and helps keep the change clear for others when looking at it months/years later.

If there's a need for style repairs throughout a file, my take is it's often good to do them in a separate change.